modules/oadp-release-notes-1-2-0.adoc

```diff
@@ -0,0 +1,81 @@
+
+[id="fixed-bugs-1-2-0_{context}"]
+== Fixed bugs
+
+The following bugs have been fixed in this release:
+
+* link:https://issues.redhat.com/browse/OADP-144[OADP-144]
+* link:https://issues.redhat.com/browse/OADP-639[OADP-639]
+* link:https://issues.redhat.com/browse/OADP-1741[OADP-1741]
+* link:https://issues.redhat.com/browse/OADP-1152[OADP-1152]
+* link:https://issues.redhat.com/browse/OADP-1143[OADP-1143]
+* link:https://issues.redhat.com/browse/OADP-1931[OADP-1931]
+* link:https://issues.redhat.com/browse/OADP-148[OADP-148]
+* link:https://issues.redhat.com/browse/OADP-1067[OADP-1067]
+* link:https://issues.redhat.com/browse/OADP-1332[OADP-1332]
+* link:https://issues.redhat.com/browse/OADP-1164[OADP-1164]
+* link:https://issues.redhat.com/browse/OADP-1105[OADP-1105]
+* link:https://issues.redhat.com/browse/OADP-2009[OADP-2009]
+* link:https://issues.redhat.com/browse/OADP-1370[OADP-1370]
+* link:https://issues.redhat.com/browse/OADP-969[OADP-969]
+* link:https://issues.redhat.com/browse/OADP-1672[OADP-1672]
+* link:https://issues.redhat.com/browse/OADP-1151[OADP-1151]
+* link:https://issues.redhat.com/browse/OADP-988[OADP-988]
+* link:https://issues.redhat.com/browse/OADP-1941[OADP-1941]
+* link:https://issues.redhat.com/browse/OADP-1830[OADP-1830]
+* link:https://issues.redhat.com/browse/OADP-1821[OADP-1821]
+* link:https://issues.redhat.com/browse/OADP-1783[OADP-1783]
+* link:https://issues.redhat.com/browse/OADP-1719[OADP-1719]
+* link:https://issues.redhat.com/browse/OADP-1833[OADP-1833]
+* link:https://issues.redhat.com/browse/OADP-1872[OADP-1872]
+* link:https://issues.redhat.com/browse/OADP-2047[OADP-2047]
+* link:https://issues.redhat.com/browse/OADP-1932[OADP-1932]
+* link:https://issues.redhat.com/browse/OADP-1844[OADP-1844]
+* link:https://issues.redhat.com/browse/OADP-1182[OADP-1182]
+* link:https://issues.redhat.com/browse/OADP-1183[OADP-1183]
+* link:https://issues.redhat.com/browse/OADP-1798[OADP-1798]
+* link:https://issues.redhat.com/browse/OADP-1726[OADP-1726]
+* link:https://issues.redhat.com/browse/OADP-821[OADP-821]
+* link:https://issues.redhat.com/browse/OADP-1833[OADP-1781]
+* link:https://issues.redhat.com/browse/OADP-697[OADP-697]
+* link:https://issues.redhat.com/browse/OADP-1281[OADP-1281]
+* link:https://issues.redhat.com/browse/OADP-1077[OADP-1077]
+* link:https://issues.redhat.com/browse/OADP-1076[OADP-1076]
+* link:https://issues.redhat.com/browse/OADP-1670[OADP-1670]
+* link:https://issues.redhat.com/browse/OADP-1307[OADP-1307]
+* link:https://issues.redhat.com/browse/OADP-1640[OADP-1640]
+* link:https://issues.redhat.com/browse/OADP-1987[OADP-1987]
+* link:https://issues.redhat.com/browse/OADP-1934[OADP-1934]
+
+[id="known-issues-1-2-0_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* Data Mover backups over `cephFS` (ODF 4.12) fails with "Asynchronous action timed out". link:https://issues.redhat.com/browse/OADP-1933[OADP-1933]
```

Comment:
@HagayVider1 - you cannot write a known issues section for release notes without reading the ticket.
There is a nice little line in it, that gives a useable section: 

OADP 1.2.x no longer requires VolSync's privledged movers and can cause errors. A user could see a failed backup that timed out with the following errors:

```
Error: container's runAsUser breaks non-root policy 
Error from server (BadRequest): container "restic" in pod "volsync-src-vsb-xxxyyy" is waiting to start: CreateContainerConfigError
```

Reply:
Then we could add something else reworked from the Q&A:

By default, OADP uses the Pod/Workloads security context to backup data using Volsync/datamover.  Also by default, `dpa.spec.features.dataMover.volumeOptions.sourceVolumeOptions.moverSecurityContext` is set to `true`.

Issue the following command to ensure the annotation is set to `false`:
```
oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'
```

> [!NOTE]
>
>  In some more advanced use case where you may want and need root escalation of privileges the option to use the annotation and setting `moverSecurityContext` to false.

Reply:
afaiu/r  moverSecurityContext defaults to false,
maybe double-check?

Reply:
https://issues.redhat.com/browse/OADP-1941?jql=text%20~%20%22moverSecurityContext%22

Reply:
> Then we could add something else reworked from the Q&A:
> 
> By default, OADP uses the Pod/Workloads security context to backup data using Volsync/datamover. Also by default, `dpa.spec.features.dataMover.volumeOptions.sourceVolumeOptions.moverSecurityContext` is set to `true`.
> 
> Issue the following command to ensure the annotation is set to `false`:
> 
> ```
> oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'
> ```
> 
> > [!NOTE]
> > In some more advanced use case where you may want and need root escalation of privileges the option to use the annotation and setting `moverSecurityContext` to false.

Should I add this to Known issue OADP-1933?

Reply:
Folks @anarnold97 for 1.2.0 `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'`   is no longer required.